opennms-doc/guide-install/src/asciidoc/text/basic-setup-opennms/rhel/step-4-change-admin-password.adoc

```diff
@@ -0,0 +1,14 @@
+
+==== Step 4: First Login and change default password
+
+After starting OpenNMS the web application can be accessed on http://<ip-or-fqdn-of-your-server>:8980/opennms.
+The default login user is _admin_ and the password is initialized to _admin_.
+
+It is highly recommended to change the default password to prevent unauthorized persons.
```

Comment:
I believe it should also be highly recommended to change the jetty configuration to use HTTPS rather than HTTP; There's not much point changing the admin password if anyone can get it with a simple MITM.